certbot-apache/certbot_apache/tests/generic_test.py

```diff
@@ -0,0 +1,106 @@
+"""Test for certbot_apache.configurator non-overridden flow"""
+import os
+import shutil
+
+import mock
+import tempfile
```

Comment:
nit: tempfile is part of the standard library and should be moved with the first group of imports.